.github/workflows/build.yml

```diff
@@ -5,40 +5,21 @@ on:
       - '.github/**'
```

Comment:
De strekking van de changes ziet er wel oke uit, beetje teveel om per regel te reviewen. Vraag me alleen of het geen major version bump moet worden, om voor bestaande zaken nog een maintenance versie te hebben.

Reply:
Ik kan een versie 1.0.0 maken maar vond zelf 0.3.0 al een grote stap. Het is in grote lijnen nog dezelfde client alleen heb ik sommige methoden wat getweakt.

Reply:
semver he, iets met breaking changes...

vooral het verplaatsen van die structs naar de model package breekt volgens mij huidige impls:

```
import (
  keyhub "github.com/topicuskeyhub/go-keyhub"
)

.... *keyhub.VaultRecord ...
```

want in de examples zie ik:
```
import (
  keyhubmodel "github.com/topicuskeyhub/go-keyhub/model"
)
```

Reply:
Euh ja, die was ik even vergeten 😄 
Dan maken we er 1.x.x van